src/assets/styleJs/events/createEvent.ts

```diff
@@ -0,0 +1,224 @@
+import {makeStyles} from "@mui/styles";
+import {borderRedius, color, font, margin, padding} from "../../style/globalVariables";
+
+const useCreateEventStyles = makeStyles({
+  rootCreateEvent: {
+    height: "100%"
+  },
+
+  eventTitleBox: {
+    display: "flex",
+    alignItems: "center",
+    width: "45%",
+    marginBottom: margin.marginBottom40
+  },
+  eventInfoBoxEvaluator: {
+    display: "flex",
+    alignItems: "center",
+    width: "40%",
+    marginBottom: margin.marginBottom25
+  },
+  eventInfoBoxEvaluatee: {
+    display: "flex",
+    alignItems: "center",
+    width: "40%",
+    marginBottom: margin.marginBottom25,
+  },
+  evaluatorCard: {
+    width: "368px",
+    height: "245px",
+    padding: `${margin.marginTop10} ${margin.marginRight15} ${margin.marginBottom40} ${margin.marginLeft15}`,
+    marginRight: margin.marginRight16,
+    marginBottom: margin.marginBottom16,
+    boxShadow: "none"
+  },
+  evaluateeCard: {
+    width: "368px",
+    height: "305px",
+    padding: `${margin.marginTop10} ${margin.marginRight15} ${margin.marginBottom40} ${margin.marginLeft15}`,
+    marginRight: margin.marginRight16,
+    marginBottom: margin.marginBottom16,
+    boxShadow: "none",
+  },
+  evaluatorCardInputBox: {
+    display: "flex",
+    justifyContent: "space-evenly",
+    flexDirection: "column",
+    height:'100%'
+  },
+  evaluateeCardInputBox: {
+    display: "flex",
+    justifyContent: "space-evenly",
+    flexDirection: "column",
+    height:'100%'
+  },
```

Comment:
do not write same thing for classes just use the same class

Reply:
done

Reply:
not fixed